examples/pattern_matching_showcase/experiment.test.sh

```diff
@@ -1,2 +0,0 @@
-mpirun -n 3 music `readlink -f run.music` TEST_MODE
-
```

Comment:
Would it produce a lot of overhead if the file experiment.test.sh was placed directly in the unittest/ folder? Probably you did that because adapting the paths in all the scripts would get really annoying, right? Semantically, I would find it better if that goes to unittest/ or a subfolder of it, but if difficult lets stick to that solution.